modules/Cockpit/assets/components.js

```diff
@@ -2783,7 +2783,8 @@ riot.tag2('field-layout', '<div class="uk-sortable layout-components {!items.len
         }
 
         if (opts.parentComponent && opts.parentComponent.options) {
-            opts = App.$.extend(true, opts.parentComponent.options, opts);
+            var clonedParentComponentOptions = App.$.extend(true, {}, opts.parentComponent.options);
+            opts = App.$.extend(true, clonedParentComponentOptions, opts);
```

Comment:
```
opts = App.$.extend(true, {}, opts.parentComponent.options, opts);
```

so there is no reason for clonedParentComponentOptions

Reply:
Maybe, I will try it on a entry I have problems with